src/lib.rs

```diff
@@ -434,6 +436,16 @@ mod contract {
         EthConnectorContract::get_instance().ft_on_transfer(&engine)
     }
 
+    #[no_mangle]
+    pub extern "C" fn get_eth_connector_paused_flags() {
+        EthConnectorContract::get_instance().get_paused_flags()
+    }
+
+    #[no_mangle]
+    pub extern "C" fn set_eth_connector_paused_flags() {
```

Comment:
I think just `set_paused_flags`

Reply:
Ok for me if this won't produce ambiguity - which is paused, etc. What do you think?

Reply:
I second `set_paused_flags` and `get_paused_flags` since nothing else will be needing it.